ansible/files/runtimes.json

```diff
@@ -205,7 +191,7 @@
                 "image": {
                     "prefix": "openwhisk",
                     "name": "action-golang-v1.11",
```

Comment:
Should we keep go-v1.11 along with go-v1.15?
Reference: [GO-EOL](https://endoflife.date/go)

Reply:
Good point.  We should drop go-v1.11 imo.